portal-frontend/src/app/features/application-details/application-details.component.html

```diff
@@ -170,7 +170,18 @@ <h4>Land Use of Adjacent Parcels</h4>
 </section>
 <section>
   <h3>6. Proposal</h3>
-  <app-nfu-details [showErrors]="showErrors" [showEdit]="showEdit" [application]="application"></app-nfu-details>
+  <app-nfu-details
+    *ngIf="application && application.typeCode === 'NFUP'"
```

Comment:
not in the scope of this MR, but it would be nice to have an enum of application types considering we will have 5+ different types